helm.rst

```diff
@@ -0,0 +1,161 @@
+Helm Charts
+===========
+
+The packaging of the CSC deployment starts with a `Helm <https://v2.helm.sh/>`_ chart. We using version 2 of Helm due the current ArgoCD restriction on the Helm version. The code for the charts are kept in the `Helm chart Github repository <https://github.com/lsst-ts/charts>`_. The next two sections will discuss each chart in detail. For a description of the APIs used, consult the `Kubernetes documentation <https://kubernetes.io/docs/reference/>`_ for the API reference. The chart sections will not go into great detail on the content of each API delivered. Each chart section will list all of the possible configuration aspects that each chart is delivering, but full use of that configuration is left to the `ArgoCD Configuration` section. For the CSC deployment, we will run a single container per pod on Kubernetes. The Kafka producers may follow a similar pattern, but they are allowed to scale beyond the single pod?
+
+Kafka Producer Chart
+--------------------
+
+While not a true control component, the Kafka producers are nevertheless an important part of the control system landscape. They have the capability to convert the SAL messages into Kafka messages that are then ingested into the Engineering Facilities Database (EFD). See :cite:`SQR-034` for more details. 
+
+The chart consists of a single Kubernetes Workloads API: Deployment. The Deployment API allows for restarts if a particular pod dies which assists in keeping the producers up and running all the time. For each producer specified in the configuration, a deployment will be created. We will now cover the configuration options for the chart.
+
+.. list-table:: Kafka Producer Chart YAML Configuration
+   :widths: 15 25
+   :header-rows: 1
+
+   * - YAML Key
```

Comment:
I didn't like it very much the way the information is displayed in the table. I wonder if it would be better to, instead, add an example yaml file.

Reply:
I can add an example to this, but the table is how most folks document their charts.